views/session/new.html

```diff
@@ -0,0 +1,22 @@
+<form class="form form--right-align" action="/session/create" method="post">
+  <fieldset>
+    <legend class="form__headline">
+      <strong>Sign in to continue :)</strong>
+      New to this site? <a class="link link--primary" href="/user/sign_up">Sign Up</a>
+    </legend>
+
+    <div class="form__group">
+      <label class="form__label" for="user_email">email: </label>
+      <input class="form__input" type="email" name="email" id="user_email" placeholder="Your email address">
+    </div>
+
+    <div class="form__group">
+      <label class="form__label" for="user_password">password: </label>
+      <input class="form__input" type="password" name="password" id="user_password" placeholder="Password">
```

Comment:
Could we use `user-password` instead of `user_password`?

Reply:
As we talked, I'll keep with snake case ✌️